cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -147,6 +148,19 @@ def _validate(self, instance_type: str, image: str):
                     FailureLevel.ERROR,
                 )
 
+        if (
+            image_info
+            and "AWS ParallelCluster AMI" in image_info.description
+            and instance_type.split(".")[0] in NVIDIA_OPENRM_UNSUPPORTED_INSTANCE_TYPES
+        ):
+            self._add_failure(
+                f"The instance type '{instance_type}' is not supported by NVIDIA OpenRM drivers. "
+                f"OpenRM can only be used on any Turing or later GPU architectures. "
```

Comment:
minor: you can remove the `f` from the lines where there are no variables to replace.